protocol/messenger.go

```diff
@@ -4192,18 +4191,22 @@ func (m *Messenger) prepareMessage(msg *common.Message, s *server.MediaServer) e
 		}
 
 		if quotedMessage.ChatMessage != nil {
+			image := quotedMessage.ChatMessage.GetImage()
 			albumID := quotedMessage.ChatMessage.GetImage().AlbumId
-			albumMessages, err := m.persistence.albumMessages(quotedMessage.LocalChatID, albumID)
-			if err != nil {
-				return err
-			}
 
-			var quotedImages = extractQuotedImages(albumMessages, s)
+			if image != nil && image.GetAlbumId() != "" {
+				albumMessages, err := m.persistence.albumMessages(quotedMessage.LocalChatID, albumID)
+				if err != nil {
+					return err
+				}
+
+				var quotedImages = extractQuotedImages(albumMessages, s)
```

Comment:
When I saw this, I thought, why not just ?

```go
quotedImages := extractQuotedImages(albumMessages, s)
```

But I believe with var you kind of force the reader for more clarity, is it intended ? If yes, then ignore my comment

Reply:
no you're right 👍 
Not my code though 😄